lib/tmuxinator/project.rb

```diff
@@ -4,11 +4,24 @@ class Project
     include Tmuxinator::Deprecations
     include Tmuxinator::WemuxSupport
 
-    attr_reader :yaml, :custom_name
+    attr_reader :yaml
+    attr_reader :force_attach
+    attr_reader :force_detach
+    attr_reader :custom_name
+
+    def initialize(yaml, options = {})
+      options[:force_attach] = false if options[:force_attach].nil?
+      options[:force_attach] = false if options[:force_attach].nil?
```

Comment:
On the duplicate lines: One should be for `:force_attach` and one for `:force_detach`.